### PR TITLE
prepare 248-rc4

### PR DESCRIPTION
**Pull request by poettering**

### CODE REVIEW

**Review by bluca — APPROVED** [review]
LGTM

**Comment by yuwata** [comment]
LGTM too.

File: NEWS [hunk 1]
```diff
@@ -437,41 +437,44 @@ CHANGES WITH 248:
...
+        Turner, Michael Marley, Michal Fabik, Michał Kopeć, Michal Koutný,
+        Michal Sekletár, Mike Gilbert, milovlad, moson-mo, Nick, nihilix-melix,
+        Oğuz Ersen, Ondrej Mosnacek, pali, Pavel Hrdina, Pavel Sapezhko, Perry
+        Yuan, Peter Hutterer, Pierre Dubouilh, Piotr Drąg, Richard Laager,
+        rnhmjoj, RussianNeuroMancer, Sebastiaan van Stijn, Sergey Bugaev,
+        shenyangyang4, simmon, Simonas Kazlauskas, Slimane Selyan Amiri, Stefan
+        Agner, Steve Ramage, Susant Sahani, Sven Mueller, Tad Fisher, Takashi
+        Iwai, Thomas Haller, Tom Shield, Topi Miettinen, Torsten Hilbrich,
+        Tyler Hicks, ulf-f, Ulrich Ölmann, Vincent Pelletier, Vinnie Magro,
+        Vito Caputo, Vlad, walbit-de, Weblate, Weblate (bot), Whired Planck,
```
Comment (dnicolodi) [inline NEWS]: Does it make sense to list Weblate and Weblate (bot) here?

Comment (poettering) [reply NEWS]: not really i guess. it' generated by a script

Comment (bluca) [reply NEWS]: Better to treat the bots nicely. You never know when a robot uprising might come :-)

**Pull request merged**